Assets/Scripts/WorkPuzzle/FallTimer.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/*
+ * This class tracks the rate in which the words are being spawned to the screen.
+ * As time passes, the rate at which words are spawned continues to speed up. 
+ */
+
+public class FallTimer : MonoBehaviour
+{
+    public WorkManager workManager;
+
+    //time between the spawning of each word
+    private float delay = 1f;
+    private float nextTime = 0f;
+
+    //boolean tracks whether or not the user has lost the game 
+    public static bool stop;
+
+    private string[] passage;
+
+    private void Start()
+    {
+        stop = false;
+        passage = PassageGenerator.GetRandomPassage();
+    }
+
+    private void Update()
+    {
+        if (Time.time >= nextTime && !stop)
```

Comment:
I think this is a good candidate for turning into a coroutine and for the level up maybe use an event or have some manager call the function. Again just to try to minimize udpate loop count.